examples/non_resizable.rs

```diff
@@ -0,0 +1,22 @@
+extern crate winit;
```

Comment:
This entire example is redundant, since we already have this: https://github.com/tomaka/winit/blob/master/examples/resizable.rs